src/index.html

```diff
@@ -2,11 +2,538 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+          href="https://fonts.gstatic.com"
+          crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
+    <title>Moyo catalog</title>
+    <link rel="stylesheet" href="./styles/style.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page">
+    <header class="header">
+      <a class="header__logo" href="#">
+        <img class="header__logo-image"
+        src="./images/logo.png"
+        alt="logo">
+      </a>
+      <nav class="header__nav">
+        <ul class="header__nav-list">
+          <li class="header__nav-item">
+            <a class="is-active header__nav-link" href="#apple">
+              Apple
+            </a>
+          </li>
+          <li class="header__nav-item">
```

Comment:
Add the blank line between the links
```suggestion
          </li>
          
          <li class="header__nav-item">
```